### PR TITLE
Configuration

### PR DESCRIPTION
**Pull request by pirc25**
### Configuration Liters. 
This project is created to learn about HTML&CSS linters
- Added the  HTML&CSS linters files: .stylelintrc.json,  .hintrc.
- Added .gitignore because It is used to git ignores the "node_modules" folder and  "test.md" file.
- added README.MD with the project information using the template.

### CODE REVIEW

**Review by Danieldotcomcoder — CHANGES_REQUESTED** [review]
Hello @pirc25, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights

- Linters are setup correctly ✔✔
- Clear and nice code ✔✔
- All Linters are passing ✔✔

## Required Changes ♻️

- Please add a descriptive Pull Request message with a title and mention the features that you implemented in a short summary. 😃✔✔


## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,6 @@
+test.md
+node_modules 
+package-lock.json
+package.json
+.hintrc
+.stylelintrc.json
```
Comment (youmari) [inline .gitignore:6]: - [ ] [OPTIONAL] please you should remove this file from the `.gitignore` file cause this file is the linter for CSS

File: README.md [hunk 2]
```diff
@@ -1 +1,28 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+> Hello Microverse Proyect.
+
```
Comment (youmari) [inline README.md:6]: - [ ] [OPTIONAL] it would be the best to add a `Getting started` header with how can someone use your project

**Review by youmari — CHANGES_REQUESTED** [review]
Hi @pirc25 , 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## Suggested changes

- [ ] please try to change the short summary as the previous reviewer asked you 
for example you can say : `Add HTML file white h1 header` and so on 
- [ ] [OPTIONAL] please try next time  to capitalize your commits as in the professional rules

_Check the comments under the review._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by smunozmo — APPROVED** [review]
Hi @pirc25, 👋 

## Highlights 👏 

- Great work using the right GitHub Flow.
- Your README looks professional, well done!
- Commits are descriptive, good work.
- Linters are working properly, great job!

Your project is complete!  There is nothing else to say other than... it's time to merge it! 🚀 
Very nice work, congratulations! 🎉 

### APPROVED! 💯  💯  💯 




Cheers and Happy coding! 🤓 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

✅  Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by pirc25** [comment]
I merged two branches with linters configuration.

**Pull request merged**